processor/spanmetricsprocessor/processor.go

```diff
@@ -379,3 +416,33 @@ func (p *processorImp) cache(serviceName string, span pdata.Span, k metricKey) {
 		p.metricKeyToDimensions[k] = buildDimensionKVs(serviceName, span, p.dimensions)
 	}
 }
+
+// copied from prometheus-go-metric-exporter
+// sanitize replaces non-alphanumeric characters with underscores in s.
+func sanitize(s string) string {
```

Comment:
Probably worth copying the [tests from prometheusexporter](https://github.com/open-telemetry/opentelemetry-collector/blob/3e138d73b3c44928045046211da421810703fd15/exporter/prometheusexporter/sanitize_test.go) too.

Reply:
Good idea!

Reply:
done